benchmark/cajita/Cajita_InterpolationPerformance.cpp

```diff
@@ -0,0 +1,344 @@
+            p2g_divergence_timer.start( ppc );
+            p2g( vector_div_p2g, position, position.size(), Spline<1>(),
+                 *scalar_halo, *scalar_grid_field );
+            p2g_divergence_timer.stop( ppc );
+
+            auto scalar_value_g2p =
+                createScalarValueG2P( scalar, -0.5 );
+            g2p_value_timer.start( ppc );
+            g2p( *scalar_grid_field, *scalar_halo, position, position.size(),
+                 Spline<1>(), scalar_value_g2p );
+            g2p_value_timer.stop( ppc );
+
+            auto scalar_gradient_g2p =
+                createScalarGradientG2P( vector, -0.5 );
+            g2p_gradient_timer.start( ppc );
+            g2p( *scalar_grid_field, *scalar_halo, position, position.size(),
+                 Spline<1>(), scalar_gradient_g2p );
+            g2p_gradient_timer.stop( ppc );
+
+            auto vector_div_g2p =
+                createVectorDivergenceG2P( scalar, -0.5 );
+            g2p_divergence_timer.start( ppc );
+            g2p( *vector_grid_field, *vector_halo, position, position.size(),
+                 Spline<1>(), vector_div_g2p );
+            g2p_divergence_timer.stop( ppc );
+        }
+
+        // Output results
+        outputResults( stream, "particle_num", particles_per_cell_dim,
+                       p2g_value_timer );
+        outputResults( stream, "particle_num", particles_per_cell_dim,
+                       p2g_gradient_timer );
+        outputResults( stream, "particle_num", particles_per_cell_dim,
+                       p2g_divergence_timer );
+        outputResults( stream, "particle_num", particles_per_cell_dim,
+                       g2p_value_timer );
+        outputResults( stream, "particle_num", particles_per_cell_dim,
+                       g2p_gradient_timer );
+        outputResults( stream, "particle_num", particles_per_cell_dim,
+                       g2p_divergence_timer );
+
+        stream << std::flush;
+    }
+}
+
+//---------------------------------------------------------------------------//
+// main
+int main( int argc, char* argv[] )
+{
+    // Initialize environment
+    MPI_Init( &argc, &argv );
+    Kokkos::initialize( argc, argv );
+
+    // Check arguments.
+    if ( argc < 2 )
+        throw std::runtime_error( "Incorrect number of arguments. \n \
+             First argument -  file name for output \n \
+             Optional second argument - run size (small or large) \n \
+             \n \
+             Example: \n \
+             $/: ./InterpolationPerformance test_results.txt\n" );
+
+    // Get the name of the output file.
+    std::string filename = argv[1];
+
+    // Define run sizes.
+    std::string run_type = "";
+    if ( argc > 2 )
+        run_type = argv[2];
+    std::vector<int> cells_per_dim = { 10, 25 };
+    std::vector<int> particles_per_cell_dim = { 1, 2, 4, 8 };
+    if ( run_type == "large" )
+        cells_per_dim = { 10, 25, 50, 100 };
```

Comment:
Let's make this the same across all grid tests - either this or Kwitae's is fine